.github/workflows/android.yml

```diff
@@ -70,6 +70,13 @@ jobs:
       - name: Add msbuild to PATH
         if: startsWith(matrix.os, 'windows')
         uses: microsoft/setup-msbuild@v1.0.2
+      
+      - name: Cache NDK
+        id: cache_ndk
+        uses: actions/cache@v2
+        with:
+          path: /tmp/android-ndk-r16b
+          key: android-ndk-${{ matrix.os }}
```

Comment:
Do you think we should encode the version number in the key too? If we ever expand our matrix to test/build on different NDKs, it might be helpful.

Reply:
Done!